libr/include/r_core.h

```diff
@@ -43,13 +44,13 @@ R_LIB_VERSION_HEADER(r_core);
 #define R_CORE_BLOCKSIZE 0x100
 #define R_CORE_BLOCKSIZE_MAX 0x3200000 /* 32MB */
 
-#define R_CORE_ANAL_GRAPHLINES          1
-#define R_CORE_ANAL_GRAPHBODY           2
-#define R_CORE_ANAL_GRAPHDIFF           4
-#define R_CORE_ANAL_JSON                8
-#define R_CORE_ANAL_KEYVALUE            16
-#define R_CORE_ANAL_JSON_FORMAT_DISASM  32
-#define R_CORE_ANAL_STAR                64
+#define R_CORE_ANAL_GRAPHLINES 1
+#define R_CORE_ANAL_GRAPHBODY 2
+#define R_CORE_ANAL_GRAPHDIFF 4
+#define R_CORE_ANAL_JSON 8
+#define R_CORE_ANAL_KEYVALUE 16
+#define R_CORE_ANAL_JSON_FORMAT_DISASM 32
+#define R_CORE_ANAL_STAR 64
```

Comment:
These changes are unrelated to the scope of the PR. Please revert them to the original.

Reply:
It wasn't following the coding style. I changed it because it showed up while running the script to check coding style.

Reply:
Changes reverted